packages/vscode-extension/package.json

```diff
@@ -767,6 +784,10 @@
         "command": "fx-extension.specifySubscription",
         "title": "%teamstoolkit.commands.specifySubscription.title%",
         "icon": "$(list-selection)"
+      },
+      {
+        "command": "fx-extension.openSurvey",
```

Comment:
I think you need to disable it in 'commandpalette' section if the command is not supposed to show in command palette.